src/routes/controllers/password.js

```diff
@@ -0,0 +1,92 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import bcrypt from 'bcryptjs';
+import models from '../../db/models';
+import sendMailer from '../../config/mailConfig';
+
+dotenv.config();
+
+const { User } = models;
+const secret = process.env.SECRET_KEY;
+const expires = { expiresIn: '2h' };
+const generateToken = payload => jwt.sign(payload, secret, expires);
+let message;
+const Password = {
+/**
+   * Collect recovery email
+   * @param {object} req
+   * @param {object} res
+   * @sends {mail} mail
+   * @returns {object} email
+   */
+  async forgotPassword(req, res) {
+    const { email } = req.body;
+
+    try {
+      const checkUser = await User.findOne({
+        where: { email },
+      });
+      if (!checkUser) {
+        message = { body: `User with this ${email} does not exist` };
+        return res.status(404).json({
+          error: message,
+        });
+      }
+      const token = generateToken({ id: checkUser.id, email });
+      const url = process.env.NODE_ENV === 'development' ? `http:localhost:3000/api/v1/reset-password?token=${token}`
```

Comment:
Lets move this to the configuration file.

Reply:
Okay, I will do this.

Reply:
@dbytecoderc